lib/util/enum_util.ex

```diff
@@ -8,7 +8,9 @@ defmodule Antikythera.EnumUtil do
   """
 
   @type context :: any
-  @type item :: any
+  @type element :: Enum.element
+
+  @not_found_error_msg "element not found"
```

Comment:
Let's change this error message to:
`"no matching element found"`

Reply:
fixed in 4547392